engine/sound/src/stb_vorbis/stb_vorbis.c

```diff
@@ -5285,10 +5285,14 @@ static void convert_channels_short_interleaved(int buf_c, short *buffer, int dat
             int v = FAST_SCALED_FLOAT_TO_INT(temp, f,15);//data[i][d_offset+j],15);
             if ((unsigned int) (v + 32768) > 65535)
                v = v < 0 ? -32768 : 32767;
-            *buffer++ = v;
+            if (buffer) { // DEFOLD: Added optional null
+               *buffer++ = v;
+            }
```

Comment:
At some point in the library history, they removed the "null" buffer compatibility.
It's a small fix that makes it a _lot_ easier to skip frames.